test/distributed/_composable/fully_shard/test_fully_shard_compile.py

```diff
@@ -0,0 +1,105 @@
+# Owner(s): ["oncall: distributed"]
```

Comment:
These two added files are a bad merge conflict I guess. They were deleted recently in main

Reply:
Darn it! Fixed.